lexik/jwt-authentication-bundle/2.3/manifest.json

```diff
@@ -5,6 +5,7 @@
     "copy-from-recipe": {
         "etc/": "%ETC_DIR%"
     },
+    "aliases": ["jwt-auth"],
```

Comment:
Indeed, the bundle is written on top of libraries like [lcobucci/jwt](https://github.com/lcobucci/jwt), it is only about authentication while jwt can be used in different contexts. 👍 for jwt-auth